tests/integration/test_elle.py

```diff
@@ -0,0 +1,293 @@
+"""
+This file is part of RedisRaft.
+
+Copyright (c) 2020-2021 Redis Ltd.
+
+RedisRaft is licensed under the Redis Source Available License (RSAL).
+"""
+import time
+import pytest as pytest
+
+from .sandbox import ElleWorker, Elle, key_hash_slot
+
+
+@pytest.mark.skipif("not config.getoption('elle_threads')")
+def test_elle_sanity(created_clusters):
```

Comment:
Is this a leftover?

Reply:
no, this is the simplest example of an elle background test (i.e. test does nothing, just "elle" stuff running in background.